sass/components/_post-right.scss

```diff
@@ -137,7 +137,17 @@
         border: none;
     }
 
+    .post-right__content--inner-wrapper {
+        position: absolute;
+        display: flex;
+        overflow: hidden;
+        flex-direction: column;
+        inset: 0;
+    }
+
     .post-create__container {
+        position: sticky !important;
```

Comment:
I know this is an ongoing issue with webapp and thats why I would like to start this campaign to remove all !importants and find way to optimise our CSS, so I would try to make this selector more specific and not use important here;.